docs/snapcraft-parts.md

```diff
@@ -187,6 +187,9 @@ required fields:
 Optional fields are:
 
  * origin-type - A hint about the type of project that contains the part. (e.g. bzr, git, tar)
+ * origin-branch - A hint about the source branch that contains the part.
+ * origin-commit - A hint about the source commit that contains the part.
+ * origin-tag - A hint about the source tag that contains the part.
```

Comment:
I don't think hint is the right word here. It seems wrong also in orgin-type. I would just remove "A hint about" In all the entries.